app/src/main/java/com/duckduckgo/app/bookmarks/ui/BookmarksActivity.kt

```diff
@@ -353,9 +359,35 @@ class BookmarksActivity : DuckDuckGoActivity() {
     }
 
     private fun deleteBookmarkFolder(bookmarkFolder: BookmarkFolder) {
-        val dialog = DeleteBookmarkFolderConfirmationFragment.instance(bookmarkFolder)
-        dialog.show(supportFragmentManager, DELETE_BOOKMARK_FOLDER_FRAGMENT_TAG)
-        dialog.listener = viewModel
+        TextAlertDialogBuilder(this)
+            .setTitle(R.string.delete)
+            .setMessage(getMessageString(bookmarkFolder))
+            .setPositiveButton(android.R.string.yes)
```

Comment:
```suggestion
            .setPositiveButton(R.string.yes)
```

better use our own values